crates/factor-key-value-azure/src/lib.rs

```diff
@@ -1,8 +1,10 @@
-use serde::Deserialize;
-use spin_factor_key_value::runtime_config::spin::MakeKeyValueStore;
-use spin_key_value_azure::{
+mod azure;
```

Comment:
Nit: the mod name `azure` feels a bit weird considering the entire crate is dedicated to Azure. I don't have a strong opinion on an alternative though. Part of me wonders if we should just merge the `azure` module with lib.rs instead of keeping them separate.

Reply:
Thoughts on this @lann? I kind of like it being in a separate module so a tie-breaking opinion would be helpful.

Reply:
`mod store`